krun.py

```diff
@@ -248,6 +250,9 @@ def main(parser):
         if rc > 0:
```

Comment:
I think I am to blame for this line, but I wonder whether `rc > 0` should be `rc != 0`?